cg/meta/workflow/mutant/quality_controller/report_generator_utils.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+from cg.io.json import write_json
+from cg.meta.workflow.mutant.constants import QUALITY_REPORT_FILE_NAME
+from cg.meta.workflow.mutant.quality_controller.models import (
+    CaseQualityResult,
+    SamplesQualityResults,
+)
+from cg.meta.workflow.mutant.quality_controller.result_logger_utils import samples_results_message
+
+
+def write_report(
+    case_path: Path,
+    case_quality_result: CaseQualityResult,
+    samples_quality_results: SamplesQualityResults,
+) -> None:
+    summary: str = get_summary(
+        case_quality_result=case_quality_result, samples_quality_results=samples_quality_results
+    )
+    report_content = {
+        "summary": summary,
+        "case": case_quality_result.model_dump(),
+        "samples": samples_quality_results.model_dump(),
+    }
+    report_file_path: Path = get_report_path(case_path=case_path)
+
+    write_json(file_path=report_file_path, content=report_content)
```

Comment:
With the changes above
```suggestion
    write_json(file_path=report_file_path, content=report_content.model_dump())
```

Reply:
Same here! This is already in place and you probably looked at an older version of the code. 😅